### PR TITLE
Set properly encrypted file size when uploading

### PR DESCRIPTION
**Pull request by kleewho**
fix: Uploading an encrypted file correctly sets the content size in the multipart body.

### CODE REVIEW

File: src/test/kotlin/com/pubnub/api/legacy/endpoints/files/SendFileTest.kt [hunk 1]
```diff
@@ -63,20 +51,61 @@ class SendFileTest : TestsWithFiles {
...
+            sendFile(channel, fileName(), inputStream).sync()
         }
 
         // then
         Assert.assertEquals(expectedResponse, result)
     }
 
     @Test
-    @Throws(InterruptedException::class, IOException::class)
+    fun spaceIdAndMessageTypeArePassedToPublishMessageFile() {
```
Comment (marcin-cebo) [inline src/test/kotlin/com/pubnub/api/legacy/endpoints/files/SendFileTest.kt:63]: Are you sure that test does what it promises? Is this possible to pass spaceId and messageType to publishMessageType?

**Review by marcin-cebo — CHANGES_REQUESTED** [review]

File: src/test/kotlin/com/pubnub/api/legacy/endpoints/files/SendFileTest.kt (hunk 1 above)
Comment (kleewho) [reply src/test/kotlin/com/pubnub/api/legacy/endpoints/files/SendFileTest.kt:63]: Nope. This shouldn't be there. It's a spoiler from other PR that will come soon ;)

Comment (marcin-cebo) [reply src/test/kotlin/com/pubnub/api/legacy/endpoints/files/SendFileTest.kt]: ok

**Review by marcin-cebo — APPROVED** [review]

**Comment by kleewho** [comment]
@pubnub-release-bot release kotlin as v7.4.1

**Pull request merged**